src/ansi.rs

```diff
@@ -1195,6 +1198,41 @@ impl StandardCharset {
     }
 }
 
+/// SCP control's first parameter which determines character path.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum ScpCharPath {
+    /// SCP's first parameter value of 0 (the default). Behavior is implementation defined.
+    ///
+    /// This value is defined by Terminals Working Group BiDi draft proposal.
+    Default,
+    /// SCP's first parameter value of 1 which sets character path to LEFT-TO-RIGHT.
+    LTR,
+    /// SCP's first parameter value of 2 which sets character path to RIGHT-TO-LEFT.
+    RTL,
+    /// SCP's first parameter with an unknown value.
+    Unknown(u16),
```

Comment:
I don't think this is actually valid, right? I don't think we should give terminal emulators the option to implement new proprietary extensions.